packages/editor/src/hooks/pattern-overrides.js

```diff
@@ -43,6 +44,7 @@ const withPatternOverrideControls = createHigherOrderComponent(
 				{ props.isSelected && isSupportedBlock && (
 					<ControlsWithStoreSubscription { ...props } />
 				) }
+				{ isSupportedBlock && <PatternOverridesBlockControls /> }
```

Comment:
It's a shame there's no way to add a condition like `( props.isSelected || props.isMultiselected )` here

Reply:
We should probably add a comment there to avoid accidental regression.